shared/ssa/qlpack.yml

```diff
@@ -1,4 +1,4 @@
 name: codeql/shared-ssa
-version: 0.0.1
+version: 0.0.1-dev
```

Comment:
This one should stay as it is, and before publishing `-dev` needs to be removed. This happens automatically for the core libraries, but will need to be manual for the shared libraries until we augment the existing automation.